apps/docs/spec/cli_v1_config.yaml

```diff
@@ -1096,6 +1112,36 @@ parameters:
       - name: 'Auth Server configuration'
         link: 'https://supabase.com/docs/reference/auth'
 
+  - id: 'edge_runtime.enabled'
```

Comment:
Actually can we move this to Edge Functions config? Currently it's under local

![Screenshot 2024-11-28 at 8 01 28 PM](https://github.com/user-attachments/assets/1211eaf4-ae98-4ca3-a141-6784b02c4a3d)